app/invocation/cache_requests_card.tsx

```diff
@@ -368,7 +371,62 @@ export default class CacheRequestsCardComponent extends React.Component<CacheReq
     ));
   }
 
+  private getCacheMetadata(scorecardResult: cache.ScoreCard.IResult) {
+    const digest = scorecardResult.digest;
+    const remoteInstanceName = this.props.model.getRemoteInstanceName();
+    rpc_service.service
+      .getCacheMetadata(
+        new cache.GetCacheMetadataRequest({
```

Comment:
nit: you can omit `new cache.GetCacheMetadataRequest(...)` since the proto library accepts a regular old object, and it will still perform the same type-checking.

Reply:
Done